mods/ra/weapons/missiles.yaml

```diff
@@ -75,93 +36,83 @@ Dragon:
 		ImpactSounds: splash9.aud
 		ValidImpactTypes: Water
 
+^AntiGroundMissile:
+	Inherits: ^AntiAllMissile
+	ValidTargets: Ground, Water
+
+^AntiAirMissile:
+	Inherits: ^AntiAllMissile
+	ValidTargets: Air
+
+Maverick:
+	Inherits: ^AntiGroundMissile
+	ReloadDelay: 30
+	Range: 9c0
+	MinRange: 3c0
+	Report: missile7.aud
+	Burst: 2
+	BurstDelay: 7
+	Projectile: Missile
+		Speed: 256
+		Inaccuracy: 512
+		CruiseAltitude: 2c0
+		RangeLimit: 14c410
+	Warhead@1Dam: SpreadDamage
+		Damage: 70
+		Versus:
+			None: 30
+			Wood: 90
+			Light: 90
+			Heavy: 100
+			Concrete: 100
+
+Dragon:
+	Inherits: ^AntiGroundMissile
```

Comment:
This also makes the warhead damage airborne actors, but I expect this is a fix to match the air->ground case.